terracumber_config/tf_files/PR-TEST-main.tf

```diff
@@ -0,0 +1,237 @@
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["server"]
+      }
+      additional_repos_only = var.ADDITIONAL_REPOS_ONLY
+      additional_repos = {
+        pull_request_repo = var.PULL_REQUEST_REPO,
+        master_repo = var.MASTER_REPO,
+        master_repo_other = var.MASTER_OTHER_REPO,
+        master_sumaform_tools_repo = var.MASTER_SUMAFORM_TOOLS_REPO,
+        test_packages_repo = var.TEST_PACKAGES_REPO,
+        non_os_pool = "http://${var.MIRROR}/distribution/leap/15.4/repo/non-oss/",
+        os_pool = "http://${var.MIRROR}/distribution/leap/15.4/repo/oss/",
+        os_update = var.UPDATE_REPO,
+        os_additional_repo = var.ADDITIONAL_REPO_URL,
+        testing_overlay_devel = "http://${var.MIRROR}/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Testing-Overlay-POOL-x86_64-Media1/",
+      }
+      image = var.IMAGE
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+      server_mounted_mirror = var.MIRROR
+    }
+    proxy = {
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["proxy"]
+      }
+      additional_repos_only = var.ADDITIONAL_REPOS_ONLY
+      additional_repos = {
+        pull_request_repo = var.PULL_REQUEST_REPO,
+        master_repo = var.MASTER_REPO,
+        master_repo_other = var.MASTER_OTHER_REPO,
+        master_sumaform_tools_repo = var.MASTER_SUMAFORM_TOOLS_REPO,
+        test_packages_repo = var.TEST_PACKAGES_REPO,
+        non_os_pool = "http://${var.MIRROR}/distribution/leap/15.4/repo/non-oss/",
+        os_pool = "http://${var.MIRROR}/distribution/leap/15.4/repo/oss/",
+        os_update = var.UPDATE_REPO,
+        os_additional_repo = var.ADDITIONAL_REPO_URL,
+        testing_overlay_devel = "http://${var.MIRROR}/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Testing-Overlay-POOL-x86_64-Media1/",
+        proxy_pool = "http://${var.MIRROR}/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/",
+        tools_update = var.OPENSUSE_CLIENT_REPO
+      }
+      image = var.IMAGE
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-minion = {
+      image = "sles15sp4o"
+      name = "min-sles15"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["suse-minion"]
+      }
+      additional_repos = {
+        tools_update = var.SLE_CLIENT_REPO,
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-sshminion = {
+      image = "sles15sp4o"
+      name = "minssh-sles15"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["suse-sshminion"]
+      }
+      additional_repos = {
+        tools_update = var.SLE_CLIENT_REPO,
+      }
+      additional_packages = [ "venv-salt-minion", "iptables" ]
+      install_salt_bundle = true
+    }
```

Comment:
On Uyuni we are currently refactoring these minions, to use Leap Minions instead of SLES Minions.
Dominik is working on that refactor.